notebooks/MAST/m_dwarf_flares_hst/m_dwarf_flares_hst.ipynb

```diff
@@ -0,0 +1,1412 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #46.            public_obs_tables = public_obs_tables[public_obs_tables['intentType'] != 'calibration']
</pre><p>This is something that you can filter directly in your search. See CAOM field descriptions for all the available fields <a href="https://mast.stsci.edu/api/v0/_c_a_o_mfields.html" target="_blank">https://mast.stsci.edu/api/v0/_c_a_o_mfields.html</a></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ScienceBetter/notebooks-stsci/pull/32/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='20'/>